### PR TITLE
ticket 25 clean up issues

### PR DESCRIPTION
**Pull request by vricci518**
Added missing new lines at end of file
Replaced \ with / in links to stylesheets
Adjusted padded per Bill suggestions

### CODE REVIEW

File: css/aboutus.css [hunk 1]
```diff
@@ -24,7 +24,7 @@
 .about p{
     width: 90%;
     margin: auto;
-    padding: 5px 0px 5px 0px;
+    padding: 5px auto;
```
Comment (billythakidd04) [inline css/aboutus.css]: I apologize, my previous comment said `5px auto` it should have been `5px 0px` thats on me

**Review by billythakidd04 — DISMISSED** [review]
one small change and thats my fault sorry.

File: css/aboutus.css (hunk 1 above)
Comment (vricci518) [reply css/aboutus.css]: file adjusted

**Review by margeschrec — APPROVED** [review]
looks good

**Pull request merged**